buzzmobile/launch/includes/visualization.launch

```diff
@@ -2,5 +2,5 @@
     <node pkg="buzzmobile" name="route_map" type="route_mapper.py"/>
     <node pkg="buzzmobile" name="state_viz" type="state_viz.py"/>
     <node pkg="rqt_gui" name="rqt_gui" type="rqt_gui" 
-          args="--perspective=$(find buzzmobile)/tools/mission_control/Default.perspective"/>
-</launch>
\ No newline at end of file
+          args="--perspective-file=$(find buzzmobile)/tools/mission_control/Test.perspective"/>
```

Comment:
Can you revert the vis to be the Default, since this is the main `visualization.launch`?